### PR TITLE
remove conditional 'view_group'

### PR DESCRIPTION
**Pull request by ShaiahWren**
Issue: AAH-1805

With the current `view_group` conditional, users without this permission cannot view the roles list nor a group detail view. This should not be the case. This pr simply removes that conditional.

### CODE REVIEW

File: CHANGES/1805.bug [hunk 1]
```diff
@@ -0,0 +1 @@
+Remove conditional `view_task`.
```
Comment (newswangerd) [inline CHANGES/1805.bug:1]: .bugfix

File: galaxy_ng/app/access_control/statements/standalone.py [hunk 2]
```diff
@@ -400,7 +400,6 @@
             "action": ["list", "retrieve"],
             "principal": "authenticated",
             "effect": "allow",
-            "condition": "has_model_perms:galaxy.view_group"
```
Comment (newswangerd) [inline galaxy_ng/app/access_control/statements/standalone.py:403]: this needs to go in insights.py too

**Review by newswangerd — COMMENTED** [review]
Can you write a quick integration test to verify that any user without permissions can read data from the roles api, but not change anything?

**Comment by ShaiahWren** [comment]
@newswangerd Can you clarify 1. which specific permissions should/not be assigned in order to manipulate roles and 1. what actions on roles should/not be allowed. Edit, delete?

**Review by newswangerd — APPROVED** [review]

**Pull request merged**